app/models/item.rb

```diff
@@ -11,7 +11,7 @@ class Item < ApplicationRecord
 
   mount_uploader :image, ImageUploader
 
-  validates :name, presence: true
+  validates :name, presence: true, uniqueness: true
```

Comment:
Only validate uniqueness within the scope of the `user_id`. Otherwise you're not allowed to create an item if another user has one with the same name.